hassio/src/update-available/update-available-card.ts

```diff
@@ -192,13 +192,7 @@ class UpdateAvailableCard extends LitElement {
                     </a>`
                   : ""}
                 <span></span>
-                <ha-progress-button
-                  .disabled=${!this._version ||
```

Comment:
Should it still be disabled if we dont have `_version`?

Reply:
nope